sway-ir/src/parser.rs

```diff
@@ -43,11 +43,12 @@ mod ir_builder {
                 }
 
             rule script_or_predicate() -> IrAstModule
-                = kind:module_kind() "{" _ configs:init_config()* _ fn_decls:fn_decl()* "}" _
+                = kind:module_kind() "{" _ configs:init_config()* _ global_consts:global_const()* _ fn_decls:fn_decl()* "}" _
```

Comment:
If we start mixing the terms "global variables/vars" and "global constants/const" it `will become very confusing :smile: 

The proposal is to stay consistent and use the term "global variable" everywhere including here in the parse. Thus, to rename:
- `global_consts` -> `global_vars`
- `global_const` -> `global_var`
- `IrAstGlobalConst` -> `IrAstGlobalVar`
- `build_global_consts_map` -> `build_global_vars_map`